### PR TITLE
feat(FTDA): Create Home Page

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Created Home Page module
- Added a background image
- Added title, description and a button to get started
### Associated Ticket Link:
- https://trello.com/c/zV7SRngG
### Screenshots / Screen Recordings: 
Screen Shot 2024-07-17 at 8 16 41 PM

### CODE REVIEW

File: pages/index.page.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import HomePageContainer from "@/modules/HomePage/containers/HomePageContainer";
 import { Inter } from "next/font/google";
 import Link from "next/link";
```
Comment (ishmam-iqbal-sazim) [inline pages/index.page.tsx:3]: Relative imports should be at the top

File: modules/HomePage/components/HomePageHero/HomePageHero.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Overlay, Container, Title, Button, Text } from "@mantine/core";
+import classes from "./HomePageHero.module.css";
+import Link from "next/link";
```
Comment (ishmam-iqbal-sazim) [inline modules/HomePage/components/HomePageHero/HomePageHero.tsx:3]: Relative imports should be at the top

**Comment by ishmam-iqbal-sazim** [comment]
Please rebase this branch off of the pointed branch and force push. Reach out if you need help.

File: pages/index.page.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import HomePageContainer from "@/modules/HomePage/containers/HomePageContainer";
```
Comment (ishmam-iqbal-sazim) [inline pages/index.page.tsx:1]: Library imports at the top. Then @ imports

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comments

File: modules/HomePage/containers/HomePageContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  Text,
+  Box,
+  BackgroundImage,
+  Flex,
+} from "@mantine/core";
+
+import classes from "./HomePageContainer.module.css";
+import { HomePageHero } from "../components/HomePageHero/HomePageHero";
+
+function HomePageContainer() {
```
Comment (ragib-hossain-sazim) [inline modules/HomePage/containers/HomePageContainer.tsx:16]: `HomePageContainer` => `HomeContainer`

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved Pending comments

**Pull request merged**